bin/gltf-pipeline.js

```diff
@@ -1,114 +1,53 @@
 #!/usr/bin/env node
 'use strict';
 var fs = require('fs');
+var argv = require('yargs').argv;
 var path = require('path');
-var argv = require('minimist')(process.argv.slice(2));
-var addDefaults = require('../').addDefaults;
-var removeUnusedImages = require('../').removeUnusedImages;
-var removeUnusedSamplers = require('../').removeUnusedSamplers;
-var removeUnusedShaders = require('../').removeUnusedShaders;
-var removeUnusedTechniques = require('../').removeUnusedTechniques;
-var removeUnusedPrograms = require('../').removeUnusedPrograms;
-var removeUnusedBuffers = require('../').removeUnusedBuffers;
-var removeUnusedBufferViews = require('../').removeUnusedBufferViews;
-var removeUnusedMaterials = require('../').removeUnusedMaterials;
-var removeUnusedSkins = require('../').removeUnusedSkins;
-var removeUnusedCameras = require('../').removeUnusedCameras;
-var removeUnusedTextures = require('../').removeUnusedTextures;
-var removeUnusedMeshes = require('../').removeUnusedMeshes;
-var removeUnusedNodes = require('../').removeUnusedNodes;
-var removeUnusedAccessors = require('../').removeUnusedAccessors;
-var removeUnused = require('../').removeUnused;
-var loadGltfUris = require('../').loadGltfUris;
-var writeGltf = require('../').writeGltf;
-var parseBinaryGltf = require('../').parseBinaryGltf;
-var addPipelineExtras = require('../').addPipelineExtras;
-var convertDagToTree = require('../').convertDagToTree;
-var combineMeshes = require('../').combineMeshes;
-var combinePrimitives = require('../').combinePrimitives;
-var removeUnusedVertices = require('../').removeUnusedVertices;
-var OptimizationStatistics = require('../').OptimizationStatistics;
 var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
 var defined = Cesium.defined;
+var writeGltf = require('../').writeGltf;
+var writeBinaryGltf = require('../').writeBinaryGltf;
+var gltfPipeline = require('../lib/gltfPipeline');
 
 if (!defined(argv._[0]) || defined(argv.h) || defined(argv.help)) {
-	var help =
+    var help =
         'Usage: node ' + path.basename(__filename) + ' [path-to.gltf or path-to.bgltf] [OPTIONS]\n' +
-        '  -o=PATH  Write optimized glTF to the specified file.\n';
+        '  -i, input=PATH Read unoptimized glTF from the specified file.\n  -b, write binary glTF file.\n' +
+        '  -s, output non-embedded files.\n  -o, output=PATH write optimized glTF to the specified file.\n';
```

Comment:
Change the -s description, maybe like this from ModelConverter:

'writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.'